src/test/mod.rs

```diff
@@ -10,6 +10,10 @@ pub struct TestEvent {
     pub correct: Option<bool>,
 }
 
+pub fn is_missed_word_event(event: &TestEvent) -> bool {
+    event.correct == Some(false) || event.correct.is_none()
```

Comment:
```suggestion
    event.correct != Some(true)
```
And this should probably be renamed to `ends_sudden_death` or something.

EDIT: I see now that this was copied from my implementation, lol.